resources/nl/cemu.po

```diff
@@ -984,929 +1181,1278 @@ msgstr ""
-msgid "Download latest community graphic packs"
-msgstr "Download de laatste community graphic packs"
+#: GeneralSettings2.cpp
+msgid "You have to restart the game in order to apply the new settings."
+msgstr ""
+"Je moet het spel opnieuw opstarten om de nieuwe instellingen toe te passen."
 
-#: GraphicPacksWindow2.cpp:344
-msgid "This graphic pack has no description"
-msgstr "Dit graphic pack heeft geen beschrijving"
+#: GeneralSettings2.cpp MainWindow.cpp VulkanRenderer.cpp
+msgid "Information"
+msgstr "Informatie"
 
-#: GraphicPacksWindow2.cpp:434 GraphicPacksWindow2.cpp:446
-msgid "Restart of Cemu required for changes to take effect"
-msgstr "Herstarten van Cemu is verplicht om deze veranderingen toe te passen"
+#: GeneralSettings2.cpp
+msgid "Select a directory containing games."
+msgstr "Kies een folder met de games erin."
 
-#: InputPanelClassic.cpp:53 InputPanelGamepad.cpp:66 InputPanelPro.cpp:47
-msgid "Left Axis"
-msgstr "Linker as"
+#: GeneralSettings2.cpp
+msgid "Online Status"
+msgstr "Online Status"
 
-#: InputPanelClassic.cpp:74 InputPanelClassic.cpp:121 InputPanelGamepad.cpp:87
-#: InputPanelGamepad.cpp:139 InputPanelPro.cpp:68 InputPanelPro.cpp:115
-#: InputPanelWiimote.cpp:151
-msgid "Deadzone"
-msgstr "Dode zone"
+#: GettingStartedDialog.cpp
+msgid ""
+"It looks like you're starting Cemu for the first time.\n"
+"This quick setup assistant will help you get the best experience"
+msgstr ""
+"Het lijkt erop dat je Cemu voor het eerst gebruikt.\n"
+"Deze snelle configuratie-assistant zal je helpen om de beste ervaring te "
+"krijgen"
 
-#: InputPanelClassic.cpp:84 InputPanelClassic.cpp:131 InputPanelGamepad.cpp:97
-#: InputPanelGamepad.cpp:149 InputPanelPro.cpp:78 InputPanelPro.cpp:125
-#: InputPanelWiimote.cpp:162
-msgid "Range"
-msgstr "Bereik"
+#: GettingStartedDialog.cpp
+msgid "mlc01 path"
+msgstr "mlc01 map"
 
-#: InputPanelClassic.cpp:99 InputPanelGamepad.cpp:117 InputPanelPro.cpp:93
-msgid "Right Axis"
-msgstr "Rechter As"
+#: GettingStartedDialog.cpp
+msgid ""
+"The mlc path is the root folder of the emulated Wii U internal flash "
+"storage. It contains all your saves, installed updates and DLCs.\n"
+"It is strongly recommend that you create a dedicated folder for it (example: "
+"C:\\wiiu\\mlc\\) \n"
+"If left empty, the mlc folder will be created inside the Cemu folder."
+msgstr ""
+"De mlc map is de hoofd map van de geëmuleerde Wii U's interne flash opslag. "
+"Het bevat je saves, geïnstalleerde updates en DLC's.\n"
+"Het is sterk aangeraden dat je een nieuwe, speciale folder voor maakt "
+"(bijvoorbeeld C:\\wiiu\\mlc\\) \n"
+"Als je het leeg houd zal de mlc map worden gemaakt in Cemu's eigen map."
+
+#: GettingStartedDialog.cpp
+msgid "Custom mlc01 path"
+msgstr "Aangepaste mlc01 map"
+
+#: GettingStartedDialog.cpp
+msgid "Select a folder"
+msgstr "Selecteer een map"
+
+#: GettingStartedDialog.cpp
+msgid "(optional)"
+msgstr "(optioneel)"
+
+#: GettingStartedDialog.cpp
+msgid "Game paths"
+msgstr "Game Folders"
```

Comment:
Instead of _folders_ you should probably use the plural form of _map_ here, or am I wrong?

Reply:
Game paths were generally a bit hard to translate. The direct translation would be "Game Paden", but (at least in my opinion) it's too ambiguous with other dutch words especially when it is used in it's singular tense, so I went with a more loose translation (it's english translation would be more like Game Folders).

For the english word folder you'd have two words that could be used. Folders sounds better then Mappen in the plural tense but map is better in the singular tense (folder isn't used as much and made some stuff seem untranslated). They're generally interchangeable though, so I thought I'd go for the more natural choice. But for the sake of consistency I'll change it to Mappen like you suggested, though I just prefer a more natural sounding translation.

Reply:
I'll just remove any mention of folder in the translation to be more consistent.